packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -2400,6 +2400,25 @@ describe('Table', () => {
                 expect(tokenValue).toBe(expectedHeight);
             });
 
+            it('adjusts height when horizontal scrollbar is shown', async () => {
```

Comment:
It's hard for me to conceptualize, but do we expect this test to pass for operating systems that use overlay scrollbars which don't take-uup space (mac os, android / ios mobile)?

Reply:
Yes, I expect this to pass with overlay scrollbars. In fact, I believe that's the state our webkit tests run in.

The part of the test that makes it pass with overlay scrollbars is using `toBeGreaterThanOrEqual` rather than `toBeGreaterThan` when comparing the token value with the calculated expected height (line 2414 of the test). In chrome & FF, the token is greater than the calculated height because of the added scrollbar, but in webkit, the token has the same value as the calculated height because it is an overlay scrollbar.

Reply:
🛳️-it!